src/kvilib/config/kvi_shortcuts.h

```diff
@@ -221,10 +221,10 @@
 	@short:
 		The list of the common keyboard shortcuts
 	@body:
-		This list contains the key sequences associated to some common actions (like selecting text, etc). Since KVIrc 4.2 the shortcuts are dinamic:
+		This list contains the key sequences associated to some common actions (like selecting text, etc.). Since KVIrc 4.2 the shortcuts are dynamic:
```

Comment:
Maybe don't add `, etc.`. You'd only really use that if you're making a list, and he only makes one point. I would just leave it at `(like selecting text`